assets/certificateTemplate.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>[TEST] Corona School Bescheinigung</title>
+</head>
+<body>
+    Name des Studenten: %NAMESTUDENT%<br/>
+    Name des Schülers: %NAMESCHUELER%<br/>
+    Heutiges Datum: %DATUMHEUTE%<br/>
+    Startdatum: %SCHUELERSTART%<br/>
+    Enddatum: %SCHUELERENDE%<br/>
+    Fächer: %SCHUELERFAECHER%<br/>
+    Text Schüler: %SCHUELERFREITEXT%<br/>
+    Stunden pro Woche: %SCHUELERPROWOCHE%<br/>
+    Stunden gesamt: %SCHUELERGESAMT%<br/>
+    Mium: %MEDIUM%<br/>
+    Screnningdatum: %SCREENINGDATUM%<br/>
```

Comment:
I guess `CERTLINK` and `CERTLINKTEXT` would be more appropriate than the unused `SCREENINGDATUM` here, so someone can also see this information during development or while testing this locally.